app/views/admin/users/_actions.html.erb

```diff
@@ -2,7 +2,7 @@
 <div class="flex relative justify-between s:justify-end gap-2 my-2 s:my-0">
   <%= link_to "Visit profile", @user.path, class: "c-cta", target: "_blank", rel: "noopener" %>
   <div class="dropdown-trigger-container">
-    <button type="button" class="c-btn c-btn--icon-alone dropdown-trigger" id="options-dropdown-trigger" aria-haspopup="true" aria-expanded="false" aria-controls="options-dropdown">
+    <button data-testid="user-options" type="button" class="c-btn c-btn--icon-alone dropdown-trigger" id="options-dropdown-trigger" aria-haspopup="true" aria-expanded="false" aria-controls="options-dropdown">
```

Comment:
A test ID attribute seemed to make the most sense here, but now I'm thinking an `aria-label="User options menu"` or something along those lines would be more appropriate.